src/pages/dataSetNotificationTemplates/form/getInitialValuesFromTemplate.ts

```diff
@@ -0,0 +1,100 @@
+export interface DataSet {
+    id: string
+    name: string
+    displayName: string
+}
+
+export interface DataSetNotificationTemplate {
+    id: string
+    code: string
+    name: string
+    displayName: string
+    subjectTemplate: string
+    messageTemplate: string
+    notificationRecipient: string
+    dataSetNotificationTrigger: string
+    relativeScheduledDays: number
+    sendStrategy: string
+    deliveryChannels: string[]
+    recipientUserGroup?: { id: string }
+    dataSets: Array<{ id: string }>
+}
+
+export type DataSetNotificationFormValues = Omit<
+    DataSetNotificationTemplate,
+    'relativeScheduledDays' | 'recipientUserGroup'
+> & {
+    relativeScheduledDays: string
+    sendEmail: boolean
+    sendSms: boolean
+    recipientUserGroup?: string
+}
+
+export const getInitialValuesFromTemplate = (
```

Comment:
Can be leverage the zod-schema and `getDefaults` like other forms instead of this?